src/EmployerRequestApprenticeTraining/SFA.DAS.EmployerRequestApprenticeTraining/Application/Commands/PostStandard/PostStandardCommand.cs

```diff
@@ -0,0 +1,11 @@
+﻿using MediatR;
+using SFA.DAS.SharedOuterApi.InnerApi.Requests.RequestApprenticeTraining;
+using System;
+
+namespace SFA.DAS.EmployerRequestApprenticeTraining.Application.Commands.PostStandard
```

Comment:
I expect this will be renamed to Cache? or something

Reply:
renamed to CacheStandardCommand